### DISCUSSION TITLE
Information security risk status in knowledgebase

### DISCUSSION THREAD

**Discussion by K8Sudo** [monarc-project/MonarcAppFO#489] [Q&A]
In the knowledgebase it is possible to change the status of information security risks between "check" and "cross". What is this function for and what does this change do?

According to our tests, this does not deactivate the risk for either existing or newly created risks.

**Comment by ruslanbaidan** [comment]
The status field (check | cross) of the Informational risks is not used for the moment.
It was used before and there is a plat to implement the risks disabling, but this is not that easy taking into account different scenarios in case if the risk is already presented in the analysis.
We are going to come back to the feature in the future.
